### ISSUE TITLE
ElasticSearch 1.7.x ?

### ISSUE THREAD

**Issue by vincenzo** [puphpet/puphpet#2517]
Hi @jtreminio, as much as I don't like it, the legacy constraints of the project I work with mean I have to work with ES 1.7.5. I don't seem to be able to install ES < 2.0; I even tried setting the version manually in the `config.yaml` and editing the `install.pp` for ES so that `repo_version` is 1.7 (as I used to do until a while ago directly in the config + a patch).

Any way around this?

**Comment by vincenzo** [comment]
Forget that; I tried again and it worked. FYI

```diff
diff --git a/puphpet/config.yaml b/puphpet/config.yaml
index c01cbac..641d7c1 100644
--- a/puphpet/config.yaml
+++ b/puphpet/config.yaml
@@ -359,7 +359,7 @@ rabbitmq:
 elastic_search:
     install: '1'
     settings:
-        version: 2.0.2
+        version: 1.7.5
         java_install: true
     instances:
         esi_fuz3u2e1bvd5:
diff --git a/puphpet/puppet/modules/puphpet/manifests/elasticsearch/install.pp b/puphpet/puppet/modules/puphpet/manifests/elasticsearch/install.pp
index 890a9a8..77a877c 100644
--- a/puphpet/puppet/modules/puphpet/manifests/elasticsearch/install.pp
+++ b/puphpet/puppet/modules/puphpet/manifests/elasticsearch/install.pp
@@ -28,7 +28,7 @@ class puphpet::elasticsearch::install
   $merged = merge($settings, {
     'java_install' => false,
     'manage_repo'  => true,
-    'repo_version' => '2.x',
+    'repo_version' => '1.7',
   })
 
   create_resources('class', { 'elasticsearch' => $merged })
```

**Comment by jtreminio** [comment]
This sounds like it should be fixed, if 1.7 works with the ES module!

**Comment by vincenzo** [comment]
It's all working for me.
